src/components/donation/Steps/DonateMethods/DonateMethods.tsx

```diff
@@ -12,41 +12,64 @@ type Props = {
   state: FormStep;
 };
 
+const tabIdx = (method?: DonationDetails["method"]) => {
+  switch (method) {
+    case "crypto":
+      return 1;
+    case "paypal":
+      return 4;
+    case "stripe":
+      return 0;
+    default:
+      return 0;
+  }
+};
+
 const tabClasses = (selected: boolean) =>
   `${
     selected
-      ? "border-orange text-orange"
-      : "border-gray-l3 dark:border-bluegray"
-  } uppercase p-2 border-b-4 hover:text-orange-l1 hover:border-orange-l1`;
+      ? "font-semibold bg-blue @md:bg-white text-white @md:text-black"
+      : "border border-prim @md:border-none"
+  } text-sm text-left p-2 @md:p-4 focus:outline-none @md:w-28 rounded @md:rounded-none`;
 
 export default function DonateMethods({ donaterConfig, state }: Props) {
   return (
     <Tab.Group
+      manual
       as="div"
-      className="grid content-start mt-2"
-      defaultIndex={state.details?.method === "crypto" ? 1 : 0}
+      className="grid @md:grid-cols-[auto_1fr]"
+      defaultIndex={tabIdx(state.details?.method)}
     >
-      <Tab.List className="grid grid-cols-2 sm:grid-cols-5 mb-6">
-        <Tab className={({ selected }) => tabClasses(selected)}>
-          Credit/Debit
-        </Tab>
+      <Tab.List className="grid grid-cols-2 gap-2 @md:gap-0 p-4 @md:p-0 @md:grid-cols-1 content-start @md:bg-blue-l4 @md:divide-y @md:divide-white">
+        <Tab className={({ selected }) => tabClasses(selected)}>Card</Tab>
         <Tab className={({ selected }) => tabClasses(selected)}>Crypto</Tab>
         <Tab className={({ selected }) => tabClasses(selected)}>Stocks</Tab>
         <Tab className={({ selected }) => tabClasses(selected)}>DAF</Tab>
         <Tab className={({ selected }) => tabClasses(selected)}>PayPal</Tab>
       </Tab.List>
-      <Tab.Panels>
+      <Tab.Panels as="div" className="p-4 @md:p-8 pt-0 @md:pt-4 ">
         <Tab.Panel>
           <Stripe
+            recipient={state.recipient}
+            step={state.step}
+            details={
+              state.details?.method === "stripe" ? state.details : undefined
+            }
             widgetConfig={donaterConfig}
             advanceOptDisplay={
               donaterConfig?.advancedOptionsDisplay ?? "collapsed"
             }
-            state={state}
           />
         </Tab.Panel>
         <Tab.Panel>
-          <Donater {...state} config={donaterConfig} />
+          <Crypto
+            recipient={state.recipient}
+            step={state.step}
+            details={
+              state.details?.method === "crypto" ? state.details : undefined
```

Comment:
if user initially goes on with payment option A, and goes back and goes on to another option B, option A data (`amount`,`currency`...) is lost